lib/tbd/psi.rb

```diff
@@ -2123,13 +2172,20 @@ def process(model = nil, argh = {})
            is[:grade       ] = true
         end
 
-        # Label edge as :rimjoist, :balcony or :balconysill if linked to:
+        # Label edge as :rimjoist, :balcony, :balconysill or :balconydoorsill if linked to:
         #   1x deratable surface
         #   1x CONDITIONED floor
         #   1x shade (optional)
         #   1x subsurface (optional)
-        balcony     = false
-        balconysill = false
+        #
+        # Despite referring to 'sill' or 'doorsill', a 'balconysill' or
+        # 'balconydoorsill' edge may instead link (rarer) cases of balcony and a
+        # fenestratio/door head. ASHRAE 90.1 2022 does not make the distinction
+        # between sill vs head when intermediatre floor, balcony and vertical
+        # fenestration meet. 'Sills' are simply the most common occurrence.
+        balcony         = false
+        balconysill     = false # vertical fenestration
+        balconydoorsill = false # opaque door
```

Comment:
- keeping **:balconysill**, even if balcony/head edge
- keeping **:balconydoorsill**, even if balcony/doorhead edge

... 90.1 doesn't make the distinction.